src/components/implementation/cbufp/default/cbufp.c

```diff
@@ -8,11 +8,9 @@
  */
```

Comment:
Shouldn't we change `implementation/cbufp/default/cbufp.c` to `implementation/cbuf/default/cbuf.c` now as we're getting rid of the distinction between cbuf/cbufp?